### ISSUE TITLE
iMac 2013 GT 755M with Dell P2715Q

### ISSUE THREAD

**Issue by nik736** [Floris497/mac-pixel-clock-patch-V2#181]
Hi there,

I have bought an iMac as the title states. Previously I was running an Hackintosh with a GTX 760 and a scaled 1440p resolution on this Dell 4k screen.

Now I want to do the same thing with the iMac but it simply won't work with more than 30Hz. I have used the CoreDisplay and Nvidia Patcher but without success, still only 30 Hz. SwitchResX only offers me a non scaled 2560x1440 resolution but that obviously looks ugly. I tried adding a custom resolution, after a reboot it says "Active" but still only 30 Hz.

When adding the same as Startup resolution and rebooting it just uses some ultra small resolution but not the one I setup in SwitchResX.

Is anyone able to help me or is this some kind of hardware limitation?

Thanks!

**Comment by Floris497** [comment]
You are using displayport? I have actually no idea about the imacs but they should work.. if it says active you should be able to select the resolution in switchresx..

**Comment by nik736** [comment]
Yep, mDP (iMac) -> Displayport (Dell)

**Comment by nik736** [comment]
Oh, I have to select the resolution I created. That makes sense now, but somehow when creating a custom resolution I don't have the input fields for "Scale to" and when selecting scaled resolution instead of custom resolution I only have the "Scale to" fields.

Is there any example custom config for what I want to achieve?

Thank you very much.

**Comment by Floris497** [comment]
Maybe your GPU doesn't allow for these scaled resolutions.. I wouldn't be able to tell.. but 4k 60Hz should be possible I think

**Comment by nesevis** [comment]
I am having the same issue with a late 2013 iMac and the `P2415Q`

**Comment by diophung** [comment]
I had similar problem with Macbook Pro 2016, using USB-C to HDMI 2.0. Cable is able to do 4k @ 60Hz but  SwitchResX can only show scaled 2560x1440 @ 30hz. 

I have to downgrade to use a scaled 2048x1080 @ 60Hz (weird ration as it is not 16:9)

**Comment by Floris497** [comment]
Did you make a custom resolution?

**Comment by diophung** [comment]
I did create a custom resolution using 2560x1440 with vertical refresh rate 60Hz, but it turns out to be blurry because it's not HiDPI.

I also tried the custom resolution 5120x2880 as I hope it will scale down to 2560x1440 as I wanted, but SwitchResX show "inactive - invalid settings".

Like I said in #217  : I could get both HiDPI 1920x1080 or 3840x2160 at 60Hz.

**Comment by Floris497** [comment]
I think you need to make a scaled resolution, first select 4k and then your scaled resolution.. but I'm not really sure..

**Comment by joevt** [comment]
Yes, you need a scaled resolution of 5120x2880. Then you'll get a 2560x1440 HiDPI resolution that uses the 4K60Hz timing.
